lib/dynamoid/config.rb

```diff
@@ -23,6 +23,7 @@ module Config
     option :use_ssl, :default => true
     option :port, :default => '443'
     option :identity_map, :default => false
+    option :convert_big_decimal, :default => false
```

Comment:
We may want to have this as a per-model or per-column option in the future, but this is good for now.